bundles/org.openhab.binding.ecotouch/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -7,7 +7,7 @@
 		<properties>
 			<property name="thingTypeVersion">1</property>
 		</properties>
-		<instruction-set targetVersion="1">
+		<instruction-set targetVersion="2">
```

Comment:
This is not expected, i think you meant to change the `thingTypeVersion` on line 8.

Reply:
Rigth - its line 8 to be changed.

It took quite a time to find out how this update process is working. At least for my chnage I could not  use "Number:Temperature" as <type> mor "Number:Dimentionless". I found that using "ecotouch:hysteresis_warmwater" is doing what I wanted to achieve. But whereas I'm now seeing my new channel for the hysteresis I still do not see the channels from the first change. Thus I'm not sure, if the first change will work as expected.
Anyhow I ended up in target version 6 or 7 and needed to revert back to 2. That's where the error happend.

Another question related to the upadtes. As interenally in my openhab teh Thing has a version of 7 or 8, is there a way to manaully set it back to 2 or do I need to remove teh binding and re-apply again?

Reply:
Not sure how you would get version 6 or 8, but maybe that is not so relevant.

For a propertest, it would be usefull to install the latest snapshot of openHAB, Install the binding , setup the `Thing` link needed channel.
Remove the binding. 
Drop the binding jar from this PR. and check if all channel upgrades are performed.

Reply:
Hello,
as mentioned I had some problems to understand what I could put in the type tag. For me using Number:Temperature or Number:Dimensionless did not. The channel had not been created. But that might be, because I tested it on live system running 4.1.2.
Anyhow, I did the test now on a dedicated test system based on 4.2-M3 and can confirm that the channel is created correctly following your suggestion for a proper test using the latest code uploaded.
![grafik](https://github.com/openhab/openhab-addons/assets/29769735/63be7155-efdd-4da0-a996-17706e3d0db1)

Best regards
Frank

Reply:
Translations can be uploaded by crowdin. 
For reference: https://www.openhab.org/docs/developer/utils/i18n.html#managing-translations